code/modules/holiday/holidays.dm

```diff
@@ -314,6 +316,30 @@
 	begin_day = 14
 	begin_month = DECEMBER
 
+/datum/holiday/thanksgiving
+	name = "Thanksgiving"
+	begin_week = 4
+	begin_month = NOVEMBER
+	begin_weekday = THURSDAY
+
+/datum/holiday/columbus
+	name = "Columbus Day"
+	begin_week = 2
+	begin_month = OCTOBER
+	begin_weekday = MONDAY
+
+/datum/holiday/mother
+	name = "Mothers' Day"
+	begin_week = 2
+	begin_month = MAY
```

Comment:
Yeah, if it's celebrated in may in every country but the uk, there's no sense to add a special exception. I know Russia at least celebrates Christmas on new years, but I don't think we have the Christmas events extend past that. (I may be wrong there)